docs/graphs/Prims-Algorithm.md

```diff
@@ -0,0 +1,108 @@
+---
+id: Prims Algorithm
+title:  Prims Algorithm
+sidebar_label:  prims-algorithm
+sidebar_position: 2
+description: Prim's algorithm is a greedy algorithm that finds the Minimum Spanning Tree (MST)
```

Comment:
replace `description: Prim's algorithm is a greedy algorithm that finds the Minimum Spanning Tree (MST)` with `description: "Prim's algorithm is a greedy algorithm that finds the Minimum Spanning Tree (MST)"`